extensions/amp-facebook-comments/1.0/amp-facebook-comments.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseElement} from './base-element';
+import {createLoaderLogo} from '../../amp-facebook/0.1/facebook-loader';
+import {dashToUnderline} from '../../../src/string';
+import {dict} from '../../../src/utils/object';
+import {getBootstrapBaseUrl, getBootstrapUrl} from '../../../src/3p-frame';
+import {isExperimentOn} from '../../../src/experiments';
+import {userAssert} from '../../../src/log';
+
+/** @const {string} */
+const TAG = 'amp-facebook-comments';
+const TYPE = 'facebook';
+
+class AmpFacebookComments extends BaseElement {
+  /** @param {!AmpElement} element */
```

Comment:
This constructor does not appear to be necessary since it only calls `super`.

Reply:
Thanks, removed!